src/components/link/link.style.ts

```diff
@@ -56,26 +98,24 @@ const StyledLink = styled.span<StyledLinkProps>`
 
       &:hover {
         cursor: pointer;
-        color: ${isSkipLink
-          ? "var(--colorsUtilityYin090)"
-          : "var(--colorsActionMajor600)"};
 
         ${StyledIcon} {
-          color: var(--colorsActionMajor600);
+          color: ${variant === "dark"
```

Comment:
I noticed in the design when the link with icon is focused, the icon colour should also change to `colorsActionMajorYin090` same as the text for both light and dark variants. Could you also include this please?

From DS:
![Screenshot 2022-09-08 at 11 48 18](https://user-images.githubusercontent.com/18368713/189103684-dfee87ed-f1b8-4362-a570-78eb0fd49847.png)

Currently:
<img width="107" alt="Screenshot 2022-09-08 at 11 48 42" src="https://user-images.githubusercontent.com/18368713/189103743-d9359dd1-1ea6-4f9e-af28-3dbf6bc852f5.png">

<img width="109" alt="Screenshot 2022-09-08 at 11 49 13" src="https://user-images.githubusercontent.com/18368713/189103852-66c2e9b0-4b36-49af-a2c3-990d40b6caf4.png">

Reply:
Bug exists in master but I'll fix in this PR